lib/constants/constants.js

```diff
@@ -10,3 +10,6 @@ export const FUND_DISTR_TYPE = {
 };
 
 export const ERROR_CODE_GENERIC = 'genericError';
+
+export const DICT_PRODUCT_IDS = 'dictProductIds';
+export const DICT_CONTRIBUTOR_NAME_TYPES = 'dictContributorNameTypes';
```

Comment:
from my perspective this should have value as `contributorNameTypes` to support current resources in various lists

Reply:
fixed